sys.py/UI/above_all_patch.py

```diff
@@ -30,8 +30,8 @@ class AboveAllPatch(object):
     _Text  =""
     _FontObj=fonts["veramono20"]
     _Parent =None
-    _Color = pygame.Color(83,83,83)
-    _ValColor= pygame.Color(0,0,255)
+    _Color = SkinManager().GiveColor('Text')
+    _ValColor = SkinManager().GiveColor('URL')
```

Comment:
I wasn't sure what to do here.
This value isn't ever used since the child class which is used in the UI overwrites the draw method.

I changed it to the URL key, but we can just remove this change if you like.
I assume you used blue so you could easily see it early in development.